versions/3.0.4.md

```diff
@@ -173,7 +173,7 @@ Note that no aspect of implicit connection resolution changes how [URLs are reso
 ### Data Types
 
 Data types in the OAS are based on the types supported by the [JSON Schema Specification Wright Draft 00](https://tools.ietf.org/html/draft-wright-json-schema-00#section-4.2).
-Note that `integer` as a type is also supported and is defined as a JSON number that is a multiple of 1.
+Note that `integer` as a type is also supported and is defined as a JSON number without a fraction or exponent part.
```

Comment:
I would prefer to backport the language about mathematical integers from #4045

Reply:
Scratch the previous comment- I did not realize this was in the Schema Object section (there are way too many PRs and issues on this at once right now).  I am strongly against this change.  The presence of "integer" as a value for `type` is standard JSON Schema and should not be repeated.  That just adds to confusion by including it in a list of exceptions _when it is not an exception._